advocacy_docs/partner_docs/ThalesCipherTrustManager/04-ConfiguringThalesCipherTrustManager.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: 'Configuration'
+description: 'Walkthrough on configuring the integration'
+---
+
+Implementing Thales CipherTrust Manager with EDB Postgres Advanced Server 15.2 with TDE requires the following components:
+
+- EDB Postgres Advanced Server version 15.2 
+- Thales CipherTrust Manager 
+
+## Prerequisites
+
+- A running EDB Postgres Advanced Server version 15.2 instance.
+- Thales CipherTrust Manager installed and deployed per your VM environment.
+
+## Configure Thales CipherTrust Manager for EDB Postgres Advanced Server
+
+After you have successfully installed Thales CipherTrust Manager you will then need to follow a few steps so you can create certificates for use.
```

Comment:
Follow  the steps outlined below to create the required certificates after you have successfully gotten Thales CipherTrust Manger up and running.